### PR TITLE
Link "virtual machine" and  "cloud computing"

### PR DESCRIPTION
**Pull request by KiranSatyaRaj**
Link to the virtual machine in the glossary when reading about it and Modify the documentation by adding a link to cloud computing to understand the example given in a better way.

Signed-off-by: KiranSatyaRaj <90622705+KiranSatyaRaj@users.noreply.github.com>

### CODE REVIEW

File: content/en/virtualization.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ refers to the process of taking a physical computer, sometimes called a server,
 and allowing it to run multiple isolated operating systems. 
 Those isolated operating systems and their dedicated compute resources (CPU, memory, and network) are 
 referred to as virtual machines or VMs. 
-When we talk about a virtual machine, we’re talking about a software-defined computer. 
-Something that looks and acts like a real computer but is sharing hardware with other virtual machines. 
+When we talk about a [virtual machine](/virtual-machine/), we’re talking about a software-defined computer. 
+Something that looks and acts like a real computer but is sharing hardware with other virtual machines.
+[Cloud computing](/cloud-computing/) model is primarily powered by virtualization technology.
```
Comment (iamNoah1) [inline content/en/virtualization.md:17]:
```suggestion
[Cloud computing](/cloud-computing/) is primarily powered by virtualization technology.
```

**Review by iamNoah1 — COMMENTED** [review]
@KiranSatyaRaj thank you for the refinement :) I like it, there is a small suggestion.

**Review by KiranSatyaRaj — COMMENTED** [review]
It is better than before, thanks for the suggestion.

**Comment by KiranSatyaRaj** [comment]
@iamNoah1 I've approved the suggestions and committed the changes, Hope it is able to merge now.

**Review by iamNoah1 — APPROVED** [review]
lgtm :)

**Review by seokho-son — APPROVED** [review]
Thanks @KiranSatyaRaj 

LGTM

**Pull request merged**